### PR TITLE
[FLINK-25589][docs][connectors] Update Chinese version of Elasticsearch connector docs

### PR DESCRIPTION
**Pull request by biyuhao**
## What is the purpose of the change

Update Chinese version document of Elasticsearch sink interface related to [FLINK-24326](https://issues.apache.org/jira/browse/FLINK-24326) (#17930)

## Brief change log

  - Translate `docs/content.zh/docs/connectors/datastream/elasticsearch.md`
  - Translate `docs/content.zh/docs/connectors/table/elasticsearch.md`
  - Trivial fix of `docs/content/docs/connectors/datastream/elasticsearch.md`

## Verifying this change

This is a document update.
Please build docs locally to verify document content and syntax.
```
cd docs
./build_docs.sh -p
```
Then you can preview `http://localhost:1313` in the local browser.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable)

### CODE REVIEW

**Comment by biyuhao** [comment]
@flinkbot run azure

File: docs/content.zh/docs/connectors/datastream/elasticsearch.md [hunk 1]
```diff
@@ -40,16 +40,12 @@ under the License.
     </tr>
   </thead>
   <tbody>
-    <tr>
-        <td>5.x</td>
-        <td>{{< artifact flink-connector-elasticsearch5 >}}</td>
-    </tr>
     <tr>
         <td>6.x</td>
```
Comment (JingGe) [inline docs/content.zh/docs/connectors/datastream/elasticsearch.md:44]:
```suggestion
       <td><= 6.3.1</td>
```

File: docs/content.zh/docs/connectors/datastream/elasticsearch.md [hunk 2]
```diff
@@ -294,131 +210,163 @@ input.addSink(esSinkBuilder.build)
 
 ### Elasticsearch Sinks 和容错
 
-启用 Flink checkpoint 后，Flink Elasticsearch Sink 保证至少一次将操作请求发送到 Elasticsearch 集群。
+默认情况下，Flink Elasticsearch Sink 不会提供任何传递健壮性的保障。
+用户可以通过配置自行启用 Elasticsearch sink 的 at-least-once 语义。
```
Comment (JingGe) [inline docs/content.zh/docs/connectors/datastream/elasticsearch.md:214]:
```suggestion
用户可以选择启用 Elasticsearch sink 的 at-least-once 语义。
```

File: docs/content.zh/docs/connectors/datastream/elasticsearch.md [hunk 3]
```diff
@@ -294,131 +210,163 @@ input.addSink(esSinkBuilder.build)
...
+
+启用 Flink checkpoint 后，Flink Elasticsearch Sink 可以保证至少一次将操作请求发送到 Elasticsearch 集群。
```
Comment (JingGe) [inline docs/content.zh/docs/connectors/datastream/elasticsearch.md:216]:
```suggestion
通过启用 Flink checkpoint ，Flink Elasticsearch Sink 可以保证至少一次将操作请求发送到 Elasticsearch 集群。
```

File: docs/content.zh/docs/connectors/datastream/elasticsearch.md [hunk 4]
```diff
@@ -294,131 +210,163 @@ input.addSink(esSinkBuilder.build)
...
+上面的示例 sink 重新添加由于资源受限（例如：队列容量已满）而失败的请求。对于其它类型的故障，例如文档格式错误，sink 将会失败。
+如若未设置 BulkFlushBackoffStrategy (或者 `FlushBackoffType.NONE`)，那么任何类型的错误都会导致 sink 失败。
 
 <p style="border-radius: 5px; padding: 5px" class="bg-danger">
 <b>重要提示</b>：在失败时将请求重新添加回内部 <b>BulkProcessor</b> 会导致更长的 checkpoint，因为在进行 checkpoint 时，sink 还需要等待重新添加的请求被刷新。
-例如，当使用 <b>RetryRejectedExecutionFailureHandler</b> 时，
-checkpoint 需要等到 Elasticsearch 节点队列有足够的容量来处理所有挂起的请求。
-这也就意味着如果重新添加的请求永远不成功，checkpoint 也将永远不会完成。
+例如，当使用 <b>FlushBackoffType.EXPONENTIAL</b> 时，
+checkpoint 会进行等待直到 Elasticsearch 节点队列有足够的容量来处理所有挂起的请求，或者达到最大重试次数。
```
Comment (JingGe) [inline docs/content.zh/docs/connectors/datastream/elasticsearch.md:355]:
```suggestion
checkpoint 会进行等待，直到 Elasticsearch 节点队列有足够的容量来处理所有挂起的请求，或者达到最大重试次数。
```

File: docs/content.zh/docs/connectors/datastream/elasticsearch.md [hunk 5]
```diff
@@ -294,131 +210,163 @@ input.addSink(esSinkBuilder.build)
...
+ * **setBulkFlushInterval(long intervalMillis)**：刷新的时间间隔（不论缓存操作的数量或大小如何）。
 
-对于 2.x 及以上版本，还支持配置如何重试临时请求错误：
+还支持配置如何对暂时性请求错误进行重试：
 
- * **bulk.flush.backoff.enable**：如果一个或多个请求由于临时的 `EsRejectedExecutionException` 而失败，是否为刷新执行带有延迟的重试操作。
- * **bulk.flush.backoff.type**：延迟重试的类型，`CONSTANT` 或者 `EXPONENTIAL`。
- * **bulk.flush.backoff.delay**：延迟重试的时间间隔。对于常量延迟来说，此值是每次重试间的间隔。对于指数延迟来说，此值是延迟的初始值。
- * **bulk.flush.backoff.retries**：延迟重试次数。
+ * **setBulkFlushBackoffStrategy(FlushBackoffType flushBackoffType, int maxRetries, long delayMillis)**：延迟重试的类型，`CONSTANT` 或者 `EXPONENTIAL`，延迟重试次数，延迟重试的时间间隔。
```
Comment (JingGe) [inline docs/content.zh/docs/connectors/datastream/elasticsearch.md:368]:
```suggestion
 * **setBulkFlushBackoffStrategy(FlushBackoffType flushBackoffType, int maxRetries, long delayMillis)**：退避延迟的类型，`CONSTANT` 或者 `EXPONENTIAL`，退避重试次数，退避重试的时间间隔。
```

File: docs/content.zh/docs/connectors/table/elasticsearch.md [hunk 6]
```diff
@@ -192,31 +177,24 @@ CREATE TABLE myUserTable (
...
       <td>Integer</td>
       <td>最大回退重试次数。</td>
     </tr>
     <tr>
       <td><h5>sink.bulk-flush.backoff.delay</h5></td>
       <td>可选</td>
-      <td style="word-wrap: break-word;">50ms</td>
+      <td style="word-wrap: break-word;">(none)</td>
       <td>Duration</td>
       <td>每次回退尝试之间的延迟。对于 <code>CONSTANT</code> 回退策略，该值是每次重试之间的延迟。对于 <code>EXPONENTIAL</code> 回退策略，该值是初始的延迟。</td>
```
Comment (JingGe) [inline docs/content.zh/docs/connectors/table/elasticsearch.md:189]:
```suggestion
      <td>每次退避尝试之间的延迟。对于 <code>CONSTANT</code> 退避策略，该值是每次重试之间的延迟。对于 <code>EXPONENTIAL</code> 退避策略，该值是初始的延迟。</td>
```

**Review by JingGe — CHANGES_REQUESTED** [review]

File: docs/content.zh/docs/connectors/datastream/elasticsearch.md [hunk 7]
```diff
@@ -40,16 +40,12 @@ under the License.
...
-        <td>5.x</td>
-        <td>{{< artifact flink-connector-elasticsearch5 >}}</td>
-    </tr>
     <tr>
         <td>6.x</td>
         <td>{{< artifact flink-connector-elasticsearch6 >}}</td>
     </tr>
     <tr>
-        <td>7 及更高版本</td>
+        <td>7.x</td>
```
[the remark this answers is not in the document]
Comment (biyuhao) [reply docs/content.zh/docs/connectors/datastream/elasticsearch.md:48]: sorry, forgot to mention that it comes from FLINK-25189
thanks for the review

**Review by JingGe — APPROVED** [review]
LGTM

**Comment by JingGe** [comment]
@flinkbot run azure

**Comment by JingGe** [comment]
@fapaul Would you like to merge this PR? Thanks.

**Pull request merged**